tempto-ldap/src/main/java/com/teradata/tempto/internal/fulfillment/ldap/DefaultLdapObjectEntryManager.java

```diff
@@ -129,15 +124,17 @@ private String addLdapDefinition(LdapObjectDefinition ldapObjectDefinition)
     }
 
     @Override
-    public List<String> addLdapDefinitions(List<LdapObjectDefinition> ldapObjectDefinitions)
+    public void addLdapDefinitions(List<LdapObjectDefinition> ldapObjectDefinitions)
     {
+        DirContext context = createContext(ldapUrl, ldapAdminDistinguishedName, ldapAdminPassword);
         try {
-            return ldapObjectDefinitions.stream()
-                    .map(this::addLdapDefinition)
-                    .collect(toList());
+            for (LdapObjectDefinition ldapObjectDefinition: ldapObjectDefinitions)
+            {
+                addLdapDefinition(ldapObjectDefinition, context);
+            }
         }
         finally {
-            closeQuietly();
```

Comment:
Oh - so it was broken before as `addLdapDefinitions` could only be called once. And on second call `context` would have already been closed.